examples/miscellanea/simpsons-monorail-map.py

```diff
@@ -0,0 +1,131 @@
+    plt.setp(plt.gcf().get_axes(), xticks=[], yticks=[])  # no axes or labels
+    ax.set_extent([-120, -72.5, 20, 50], crs=ccrs.Geodetic())  # center on USA
+
+    # Plot only the USA landmass, in a fawn colour with a thin black border
+    shpfilename = shpreader.natural_earth(
+        resolution="110m", category="cultural", name="admin_0_countries"
+    )
+    countries = shpreader.Reader(shpfilename).records()
+    usa_border = [
+        country.geometry
+        for country in countries
+        if (country.attributes["NAME"] == "United States of America")
+    ]
+    ax.add_geometries(
+        usa_border,
+        GEOM_PROJ,
+        facecolor="#C39B6A",
+        edgecolor="black",
+    )
+
+    # From/see example:
+    # https://scitools.org.uk/cartopy/docs/latest/gallery/
+    # scalar_data/eyja_volcano.html
+    # Now add the location labels
+    #
+    # Use the cartopy interface to create a matplotlib transform object
+    # for the Geodetic coordinate system. We will use this along with
+    # matplotlib's offset_copy function to define a coordinate system which
+    # translates the text by 25 pixels to the left.
+    geodetic_transform = GEOM_PROJ._as_mpl_transform(ax)
+    text_transform = offset_copy(geodetic_transform, units="dots", x=-25)
+    for loc_name, loc_details in LOCATIONS_TO_PLOT.items():
+        loc_coords, rel_text_pos, text_rot = loc_details
+        ax.plot(
+            *loc_coords,
+            marker="o",
+            color="black",
+            markersize=6,
+            transform=GEOM_PROJ,
+        )
+
+        # Adjust position of location name text relative to location marker
+        text_loc_coords = (
+            loc_coords[0] + rel_text_pos[0],
+            loc_coords[1] + rel_text_pos[1],
+        )
+        # Text in uppercase, very bold handwriting-like font, as per the
+        # screen grab of the map from the show
+        ax.text(
+            *text_loc_coords,
+            loc_name.upper(),
+            verticalalignment="center",
+            horizontalalignment="left",
+            transform=text_transform,
+            fontname="Charcoal",  # ensure you have this font available
+            fontweight="black",
+            fontsize=28,
+            rotation=text_rot,  # slightly wonky text for handwritten effect
+        )
+
+    leg_text = (
+        "Pre-Springfield Lanley\nMonorail locations in TV's\nThe Simpsons\n"
+        "(recreation of map at\nsimpsons.fandom.com/\nwiki/Brockway)"
+    )
+
+    # Add the 'compass' legend
+    ax.text(
+        0.14,
+        0.10,
+        leg_text,
+        transform=ax.transAxes,
+        fontsize=11,
+        horizontalalignment="center",
+        verticalalignment="center",
+        style="italic",
+        bbox=dict(facecolor="#A5B5CE"),
+    )
+
+    # Make border symmetrical since default 'rc' file has asymmetric side pad
+    fig.tight_layout()
+    fig.subplots_adjust(left=0.035, bottom=0.035, right=0.965, top=0.965)
```

Comment:
Do you want to call `add_axes()` above to make an explicit placement in figure space?

Reply:
I will try playing around with use of `add_axes`, though from your previous comment it seems we don't want multiple axes so I am a bit confused as to how that fits with said plan - unless I use that to create the one and only axis. Would be useful if you can clarify?

Reply:
Sorry for the confusion, I was making various comments depending on which direction you were going to go. I agree with only one `axes`, how you put it in depends on what you want to do with it. If you want to explicitly place it or for it to take up the entire figure then I would manually place it with `ax = fig.add_axes([0, 0, 1, 1], projection=GEOM_PROJ)` (or subtract the margins you have above as necessary)